wrappers/java/wrapper

```diff
@@ -33,23 +33,25 @@ wrapper_dir=$(dirname "$0")
 cd "$wrapper_dir"
 
 if [ "$action" == "version" ]; then
-	exec ant -S -q version
+	exec mvn -S -q version
 elif [ "$action" == "compile" ]; then
 	if [[ -z "$module" ]]; then
 		echo "Syntax error"
 		echo "$usage"
 		exit 1
 	fi
-	#echo "compiling:" ant "-Dorg.ensembl.hive.module=$module" compile_wrapper
-	exec ant "-Dorg.ensembl.hive.module=$module" compile_wrapper
+	#echo "compiling:" mvn package
+	exec mvn package
```

Comment:
`mvn package` is not the same as what `ant` was doing. This command should compile the wrapper _and_ run the `org.ensembl.hive.CompileWrapper` module with `$module` (the name of the Runnable that eHive wants to use